snare/tests/test_cloner_init.py

```diff
@@ -1,18 +1,45 @@
-import unittest
-import sys
-from snare.cloner import Cloner
 import shutil
+import sys
+import unittest
+from unittest.mock import patch
+
+from snare.cloner import BaseCloner, CloneRunner, HeadlessCloner, SimpleCloner
 
 
 class TestClonerInitialization(unittest.TestCase):
     def setUp(self):
         self.root = "http://example.com"
         self.max_depth = sys.maxsize
         self.css_validate = False
-        self.handler = Cloner(self.root, self.max_depth, self.css_validate, default_path="/tmp")
+        self.target_path = None
+        self.handler = None
+
+    def test_base_cloner_init(self):
```

Comment:
seems rather useless

Reply:
do we really need these init tests? what it brings? how it helps to prevent errors?

Reply:
I have now kept only the test where there is a initialization error and have removed the rest. On this note, what do you think about the tests for the run method?